src/viur/core/skeleton.py

```diff
@@ -1451,41 +1451,30 @@ def handleEntry(cls, skel: SkeletonInstance, customData: Dict[str, str]):
     @classmethod
     def handleFinish(cls, totalCount: int, customData: Dict[str, str]):
         QueryIter.handleFinish(totalCount, customData)
+        if not customData["notify"]:
+            return
         try:
-            if customData["notify"]:
-                txt = f"Rebuild search index finished for {customData['module']}\n\n" \
-                      f"ViUR finished to rebuild the search index for module {customData['module']}.\n" \
-                      f"{totalCount} records updated in total on this kind."
-                email.sendEMail(dests=customData["notify"], stringTemplate=txt, skel=None)
-        except:  # OverQuota, whatever
-            pass
+            txt = f"Rebuild search index finished for {customData['module']}\n\n" \
```

Comment:
I would like to include the project_id here as well!
```suggestion
            txt = f"""{conf["viur.instance.project_id"]}: Rebuild search index finished for {customData["module"]}\n\n""" \
```